Makefile

```diff
@@ -57,17 +57,19 @@ check: gopath
 
 .PHONY: lint
 lint: gopath
-	@gometalinter.v1 --deadline=10m --tests --vendor --disable-all \
+	@extra_args= ;\
+	[ -z "${CYCLO_MAX}" ] || extra_args="--enable=gocyclo --cyclo-over=${CYCLO_MAX}" ;\
+	gometalinter.v1 --deadline=10m --tests --vendor --disable-all \
 	--enable=misspell \
 	--enable=vet \
 	--enable=ineffassign \
 	--enable=gofmt \
-	--enable=gocyclo --cyclo-over=10 \
 	--enable=golint \
 	--enable=deadcode \
 	--enable=varcheck \
 	--enable=structcheck \
 	--enable=unused \
+	$$extra_args \
```

Comment:
The proposed change could be written more succulently. If the intention is to use a Makefile variable, then use a makefile construct to generate the line. If the intention is to use a shell variable, then use parameter expansion, e.g.

            ${CYCLO_MAX:+ --enable-gocyclo --over-cyclo=$CYCLO_MAX} \
(double the dollars for inside a Makefile of course)

Reply:
Changed to use ":+". Thanks for the tip.

Reply:
Nice!